timemachine/cpp/src/kernels/k_nonbonded_deterministic.cuh

```diff
@@ -114,26 +182,28 @@ void __global__ k_nonbonded_jvp(
     Surreal<RealType> g_sigj(0.0, 0.0);
     Surreal<RealType> g_epsj(0.0, 0.0);
 
+    RealType inv_cutoff = 1/cutoff;
```

Comment:
I would use Static_cast<RealType> 1.0/cutoff if just to avoid compiler earnings.

Reply:
If you want to be really fancy see rd_cast in the rdkit sources.  In debug mode it throws an error on overflow.

Reply:
I'm using the templated OtherType method we spoke about so there shouldn't be any warnings.